detox/test/src/app.js

```diff
@@ -98,6 +100,7 @@ class example extends Component {
             throw new Error('Simulated Crash')
           })}
           {this.renderScreenButton('Shake', Screens.ShakeScreen)}
+          {this.renderScreenButton('Launch Args', Screens.LaunchArgsScreen)}
```

Comment:
This should only be rendered on Android or it will crash on iOS.

Reply:
I changed the test itself to e2e-only, but now also made this render only on Android 👍

Reply:
👍 Thanks